src/keyinput.js

```diff
@@ -1,5 +1,13 @@
 import { charIsDigit } from "./utils/math.js";
 
+export const button = Object.freeze({
```

Comment:
This should probably be `Button` (more like an enum that a variable)
```suggestion
export const Button = Object.freeze({
```